components/places/src/db/schema.rs

```diff
@@ -245,9 +245,33 @@ pub fn init(db: &PlacesDb) -> Result<()> {
     Ok(())
 }
 
+/// Helper for upgrade. Intended use:
+///
+/// ```rust,ignore
+/// from = migration(db, from, 2, &[stuff, to, migrate, version2, to, version3])?;
+/// from = migration(db, from, 3, &[stuff, to, migrate, version3, to, version4])?;
+/// from = migration(db, from, 4, &[stuff, to, migrate, version4, to, version5])?;
+/// ```
+///
+/// Etc.
+fn migration(db: &PlacesDb, cur_version: i64, from: i64, stmts: &[&str]) -> Result<i64> {
```

Comment:
We might eventually want this to run more complicated migration functions, not just statements, but this looks great now. We can generalize it later if we need.